src/styles/sass/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star--active {
```

Comment:
`stars` block should have modificator instead, like `stars--4`. Just reuse code from `layout_stars` task